apple/RNSVGRenderable.mm

```diff
@@ -239,134 +261,42 @@ - (void)renderTo:(CGContextRef)context rect:(CGRect)rect
   [self beginTransparencyLayer:context];
 
   if (self.mask) {
-    // https://www.w3.org/TR/SVG11/masking.html#MaskElement
-    RNSVGMask *_maskNode = (RNSVGMask *)[self.svgView getDefinedMask:self.mask];
     CGRect bounds = CGContextGetClipBoundingBox(context);
     CGSize boundsSize = bounds.size;
     CGFloat height = boundsSize.height;
     CGFloat width = boundsSize.width;
-    CGFloat scale = 0.0;
-#if TARGET_OS_OSX
-    scale = [[NSScreen mainScreen] backingScaleFactor];
-#else
-    if (@available(iOS 13.0, *)) {
-      scale = [UITraitCollection currentTraitCollection].displayScale;
-    } else {
-#if !TARGET_OS_VISION
-      scale = [[UIScreen mainScreen] scale];
-#endif
-    }
-#endif // TARGET_OS_OSX
-    NSUInteger iheight = (NSUInteger)height;
-    NSUInteger iwidth = (NSUInteger)width;
-    NSUInteger iscale = (NSUInteger)scale;
-    NSUInteger scaledHeight = iheight * iscale;
-    NSUInteger scaledWidth = iwidth * iscale;
-    NSUInteger npixels = scaledHeight * scaledWidth;
     CGRect drawBounds = CGRectMake(0, 0, width, height);
 
-    // Allocate pixel buffer and bitmap context for mask
-    NSUInteger bytesPerPixel = 4;
-    NSUInteger bitsPerComponent = 8;
-    NSUInteger bytesPerRow = bytesPerPixel * scaledWidth;
-    CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
-    UInt32 *pixels = (UInt32 *)calloc(npixels, sizeof(UInt32));
-    CGContextRef bcontext = CGBitmapContextCreate(
-        pixels,
-        scaledWidth,
-        scaledHeight,
-        bitsPerComponent,
-        bytesPerRow,
-        colorSpace,
-        kCGImageAlphaPremultipliedLast | kCGBitmapByteOrder32Big);
-    CGContextScaleCTM(bcontext, iscale, iscale);
+    // Render content of current SVG Renderable to image
+    CGImageRef currentContent = renderToImage(self, boundsSize, rect, nil);
+    CIImage *contentSrcImage = [CIImage imageWithCGImage:currentContent];
 
+    // https://www.w3.org/TR/SVG11/masking.html#MaskElement
+    RNSVGMask *_maskNode = (RNSVGMask *)[self.svgView getDefinedMask:self.mask];
```

Comment:
why `_maskNode` and not just `maskNode`? Is it somehow messing with some other variable?